### DISCUSSION TITLE
Напоминалка с краткой сводкой о сегодняшнем дне

### DISCUSSION THREAD

**Discussion by reinbowARA** [0niel/university-app#221] [Идеи]
Сдалать напоминалку. Т.е. сделать так чтобы присылалось увидомление в котором была краткая информация о сегоднешнем дне: какая идет неделя(четная/нечетная), какие сегодня пары и в каких они кобинетах и когда они начинаются и заканчиваются

**Comment by ValeryVerkhoturov** [comment]
Поздно пить боржоми, когда почки отказали
